### ISSUE TITLE
amp platform start command fails to execute

### ISSUE THREAD

**Issue by neha-viswanathan** [appcelerator-archive/amp#472]
After pulling AMP images using the command `amp platform pull`, executing `amp platform start -v` fails. 

```
│[15:16] nviswanathan@ITEM-AX40155:amp (master u=) $ amp platform start -v
Warning: no valid configuration file (.amp.yaml) found in home or current directory
Starting AMP platform
Create network amp-infra
Start error: Error response from daemon: datastore for scope "global" is not initialized
Service influxdb already stopped
Service registry already stopped
Service kapacitor already stopped
Service telegraf-haproxy already stopped
Service haproxy already stopped
Service amp-log-worker already stopped
Service grafana already stopped
Service amplifier already stopped
Service etcd already stopped
Service amp-ui already stopped
Service nats already stopped
Service elasticsearch already stopped
Service amp-agent already stopped
Service telegraf-agent already stopped
```

However, executing `docker swarm init` before running `amp platform start -v` command fixes this issue.

**Comment by freignat91** [comment]
yes, amp commands can work only on a running swarm platform, like the previous script 'swarm' we used to start amp.

**Comment by neha-viswanathan** [comment]
But with the previous swarm script, there was no need to explicitly execute `docker swarm init` before doing a `swarm start`
As per the tests in #455 `docker swarm init` is not executed before starting the swarm, which why I had this error showing up.

`make install`
stop amp: `amp platform stop`
remove all images and containers
execute: `amp platform pull -v` : pull all images needed for amp (-v: to have details)
execute: `amp platform monitor` in another console to monitor amp
execute: `amp platform start -v` in the first console
execute: `amp platform status` it returns 'running'
execute: `make test`
execute: `amp platform stop`

**Comment by freignat91** [comment]
yes, you're right, sorry to have closed this issue too quickly. Looking closer, swarm script initialized swarm. It's not the role of amp commands right now. Not the same responsibility.
Swarm cluster should be ready before executing any amp commands. For instance, using the installation agent/scripts. like the Nicolas's project "amp-swarm-deploy".
For local dev environment, I propose to update the documentation adding the fact that a `docker swarm init` is needed before using amp. What do you think?

**Comment by bquenin** [comment]
I agree with Francois. `amp` commands should assume there is already a swarm cluster up and running (regardless it's a local swarm like we have in development mode, or a multi-node swarm cluster deployed with `amp-swarm-deploy` for instance) .
We should definitely update the documentation about this.

**Comment by bquenin** [comment]
Fixed by #483
